src/templates/article.js

```diff
@@ -125,21 +124,27 @@ const Article = props => {
         },
         ...rest
     } = props;
-    const slugWithAllSlashes = addLeadingSlashIfMissing(
-        addTrailingSlashIfMissing(slug)
-    );
-    const meta = { authors, slug: slugWithAllSlashes, title };
```

Comment:
Did we mean to make this change? We want the leading and trailing slash for ratings but that seems to have been removed?

Reply:
We just used slug on 149 line. And slugWithAllSlashes was unused.

Reply:
We check for `slug` from `meta` [in this line](https://github.com/mongodb/devhub/blob/staging/src/components/dev-hub/feedback/feedback-container.js#L91) for the feedback. I didn't see if we added `slug` to `meta` anywhere else in the component?

Reply:
Oh, I see, We use this logic for feedback modal. Thanks @jestapinski!